cmd/kubeadm/app/cmd/phases/util_test.go

```diff
@@ -44,8 +44,8 @@ func TestSetKubernetesVersion(t *testing.T) {
 		},
 		{
 			name:   "any other version is skipped",
-			input:  "v1.12.0",
-			output: "v1.12.0",
+			input:  "v1.13.0",
+			output: "v1.13.0",
 		},
```

Comment:
should we use constants.CurrentKubernetesVersion here?

Reply:
I guess that I no need to change this test